esmgrids/cice_grid.py

```diff
@@ -65,66 +81,95 @@ def fromfile(cls, h_grid_def, mask_file=None, description="CICE tripolar"):
             description=description,
         )
 
-    def write(self, grid_filename, mask_filename):
+    def create_2d_nc_var(self, nc, name):
```

Comment:
```suggestion
    def _create_2d_nc_var(self, nc, name):
```

Common convention is to prefix private functions (i.e. not part of the public API) using a `_`

Reply:
Also, you've imported `netcdf4` under the alias `nc`. It's not a good idea to use `nc` as a variable name.